src/zampy/datasets/era5.py

```diff
@@ -30,8 +30,8 @@ class ERA5(Dataset):  # noqa: D101
 
     raw_variables = (
         Variable(name="mtpr", unit=unit_registry.kilogram_per_square_meter_second),
-        Variable(name="strd", unit=unit_registry.joule_per_square_meter),
-        Variable(name="ssrd", unit=unit_registry.joule_per_square_meter),
+        Variable(name="strd", unit=unit_registry.watt_per_square_meter),
```

Comment:
See my other comment on the conversion. I think it's better to keep the raw_variable units aligned with the actual units in the data.

Reply:
It makes sense. It is changed back to `joule_per_square_meter`.